vm/qemu/qemu.go

```diff
@@ -154,13 +158,14 @@ var archConfigs = map[string]*archConfig{
 		Qemu:      "qemu-system-ppc64",
 		TargetDir: "/",
 		QemuArgs:  "-enable-kvm -vga none",
+		NetDev:    "virtio-net-pci",
```

Comment:
@ajdlinux Is it the right one for ppc64? Will it work? What do you use? Not sure what was the default. I see that ppc/mips qemu still emulates e1000, but the virtio one should be faster/newer/better.

Reply:
On ppc64, the default nic type is spapr-vlan/ibmveth (which for, uh, reasons, doesn't actually show up in the net nic supported list), I don't see any reason why virtio-net-pci shouldn't work though